docs/user/whats-new.asciidoc

```diff
@@ -1,12 +1,255 @@
+
+[role="screenshot"]
+image::images/highlights-lens-anomaly.png[Create anomaly detection job option and transition to create job wizard]
+
+
+[float]
+==== Accuracy mode in Lens
+When creating *Lens* visualizations with the *Top values* function,
+you can now enable accuracy mode. This mode improves the accuracy of results for
+high-cardinality data by using more resources in your {es} cluster.
+For information, check {kibana-ref}/lens.html[Lens].
+
+[role="screenshot"]
+image::images/highlights-lens-accuracy.png["Accuracy mode in Lens", width=50%]
+
+
+
+[float]
+==== Synchronized tooltips in Lens
+
+To make it easier to compare multiple time series visualizations,
+dashboards now include a *Sync tooltips across panels* option.
+When enabled and you hover your cursor over a chart, the tooltips on all
+other related dashboard charts automatically appear at the same data point or moment in time.
+For information, check {kibana-ref}/dashboard.html[Dashboard].
+
+[role="screenshot"]
+image::images/highlights-lens-sync.png[Sync tooltip across panels option]
+
+
+[float]
+==== Collapse break down by
+
+When creating *Lens* tables and XY visualizations, you can now collapse the breakdown dimension,
+allowing you to summarize aggregated data.
+For information, check {kibana-ref}/lens.html#create-custom-tables[Create custom tables].
+
+[float]
+[[highlights-8.3-maps]]
+=== Enhancements to Maps
+
+[float]
+==== Language selector
+
+You can now select Arabic, German, English, Spanish,
+French, Hindi, Italian, Japanese, Portuguese, Russian, and Chinese for country
+and region labels. By default, new maps use the {kib}
+{kibana-ref}/i18n-settings-kb.html#general-i18n-settings-kb[locale setting] to determine
+the language for basemap labels.
+
+[role="screenshot"]
+image::images/highlights-maps-labels.gif[Menu for selecting a language for country and region labels]
+
+[float]
+==== Vector tiles and joins
+Are you using vector tiles? Good news. You are now able to make
+{kibana-ref}/terms-join.html[joins].
+
+[role="screenshot"]
+image::images/highlights-maps-vector-tiles.png[Joins for vector tiles in Maps]
+
+[float]
+[[highlights-8.3-security]]
+=== Improved Content Security Policy
+
+{kib} now has experimental support for a more restrictive
+https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP[Content Security Policy].
+Specifically, we removed the need for script-src ‘unsafe-eval’,
+which provides an additional level of protection against Cross-Site Scripting (“XSS”) attacks.
+
+To enable this feature, configure` csp.disableUnsafeEval: true` in your kibana.yml.
+
+[role="screenshot"]
+image::images/highlights-security.png[Setting for a more restrictive Content Security Policy]
+
+[float]
+[[highlights-8.3-ml]]
+=== Trained models test in Machine Learning
+
+You can now visualize machine learning trained model output.
+A new Test model action in *Machine Learning > Trained Models* opens up a
```

Comment:
Maybe it's just me and capitals having a dance today, but I'd remove this one too:
```suggestion
A new test model action in *Machine Learning > Trained Models* opens up a
```